pandas/formats/format.py

```diff
@@ -670,20 +670,28 @@ def _format_col(self, i):
                             float_format=self.float_format, na_rep=self.na_rep,
                             space=self.col_space, decimal=self.decimal)
 
-    def to_html(self, classes=None, notebook=False):
+    def to_html(self, classes=None, notebook=False, border=None):
         """
         Render a DataFrame to a html table.
 
         Parameters
         ----------
+        classes : str or list-like
+            classes to include in the `class` attribute of the opening
+            ``<table>`` tag, in addition to the default "dataframe".
         notebook : {True, False}, optional, default False
             Whether the generated HTML is for IPython Notebook.
+        border : int
+            A ``border=border`` tag is included in the opening ``<table>`` tag.
```

Comment:
I'm not really up to date on HTML nomenclature, but I think `border` is probably an attribute rather than a tag?